pkg/input/tailer/tailer.go

```diff
@@ -29,6 +29,7 @@ type Tailer struct {
 	path string
 	file *os.File
 
+	msgOffset         int64
```

Comment:
could we rename lastOffset and msgOffset? maybe `readOffset` and `decodedOffset`?